server/test/unit/vitalsTest/checkHeartbeatTest.js

```diff
@@ -0,0 +1,551 @@
+// Third-party dependencies
+const { expect, use } = require('chai')
+const { describe, it } = require('mocha')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const rewire = require('rewire')
+
+// In-house dependencies
+const { helpers } = require('brave-alert-lib')
+const Location = require('../../../Location')
+const Client = require('../../../Client')
+const db = require('../../../db/db')
+const redis = require('../../../db/redis')
+const RADAR_TYPE = require('../../../RadarTypeEnum')
+
+const vitals = rewire('../../../vitals')
```

Comment:
@mariocimet , check this library out, it might be helpful for you too: https://www.npmjs.com/package/rewire

Reply:
hah, I had run into this problem before with the state machine, and my solution at the time was to factor out local helpers into another file and import them so they could be stubbed. This feels more elegant